### ISSUE TITLE
[SUGGESTION] WZS: Replace beer

### ISSUE THREAD

**Issue by Loickun** [Azurblau/Zombies.Zone#114]
Beer is utterly complete useless as an Item. No point it having as an item at all since all it does it gives you dizzy effect and give you a little amount of stammina back. Wich the dizzy effect completly defeats its purpose to be a first aid kind of item when your stammina is low. You rather use adrenalin for that tbh or buy a trait in the beginning or midgame to have stammina regen.

About the idea:
Replace the name beer with "Bleach" and make it damage your health but gives a fair amount of Stammina back (like 40 points of stammina maybe).  Makes no sense? Well getting dizzy from 1 freaking beer makes no sense..

**Comment by mikan1216** [comment]
Yes

**Comment by HoneyBadger1337** [comment]
I agree with removing beer, it's only good to keep a hold off so you don't get it twice (Yes i'm aware it also gives you stamina, but having a screen that is disorienting isn't exactly gonna do you much good, is it?)

However, I don't agree with the bleach idea, the last thing I would really want to see is 10 kids on the server going "LOOK AT ME I'M DRINKING BLEACH XDDDDDDD AREN'T I EDGY???"

Just remove beer.

**Comment by Loickun** [comment]
What will it be replaced with then?
